create/templates/checkout_post_purchase/next-steps.txt

```diff
@@ -0,0 +1,11 @@
+
+🔭 > If this is the first time you're testing a Post Purchase extension,
+🔭 > please install the Chrome or Firefox browser extension from
+🔭 > https://github.com/Shopify/post-purchase-devtools/releases.
+
+🔭 > Once installed, simply enter your extension URL
+🔭 > {{.IntegrationContext.PublicUrl}}/extensions/{{.Extension.UUID}}.
+
+🔭 > If you prefer to test your extension without using our browser extension,
```

Comment:
As we discussed, this seems to not work for Post Purchase so we should remove this. Can we also remove it from the [html page](https://github.com/Shopify/shopify-cli-extensions/blob/0dc47a6dfb024dd7e4900b7764aa5b67ff9c4b1e/api/root/templates/checkout_post_purchase/index.html.tpl#L38)?